ui/src/app/views/workspace-edit-underserved-population/component.tsx

```diff
@@ -0,0 +1,316 @@
+      {id: UnderservedPopulationEnum.SEXUALORIENTATIONLESBIAN, label: 'Lesbian'},
+      {id: UnderservedPopulationEnum.SEXUALORIENTATIONBISEXUAL, label: 'Bisexual'},
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONPOLYSEXUALOMNISEXUALSAPIOSEXUALORPANSEXUAL,
+        label: 'Polysexual, omnisexual, sapiosexual or pansexual'
+      },
+      {id: UnderservedPopulationEnum.SEXUALORIENTATIONASEXUAL, label: 'Asexual'},
+      {id: UnderservedPopulationEnum.SEXUALORIENTATIONTWOSPIRIT, label: 'Two-Spirit'},
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONFIGURINGOUTSEXUALITY,
+        label: 'Have not figured out or are in the process of figuring out their sexuality'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONMOSTLYSTRAIGHT,
+        label: 'Mostly straight, but sometimes attracted to people of their own sex'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTTHINKOFHAVINGSEXUALITY,
+        label: 'Does not think of themselves as having sexuality'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTUSELABELS,
+        label: 'Does not use labels to identify themselves'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTKNOWANSWER,
+        label: 'Does not know the answer'
+      }
+    ]
+  },
+  {
+    title: 'Geography',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.GEOGRAPHYURBANCLUSTERS,
+        label: 'Urban clusters (2,500-50,000 people)'
+      },
+      {
+        id: UnderservedPopulationEnum.GEOGRAPHYRURAL,
+        label: 'Rural (All population, housing and territory not included within an urban area)'
+      }
+    ]
+  },
+  {
+    title: 'Disability',
+    categories: [
+      {id: UnderservedPopulationEnum.DISABILITYPHYSICAL, label: 'Physical Disability'},
+      {id: UnderservedPopulationEnum.DISABILITYMENTAL, label: 'Mental Disability'}
+    ]
+  },
+  {
+    title: 'Access to care',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.ACCESSTOCARENOTPASTTWELVEMONTHS,
+        label: 'Have not had a clinic visit in the past 12 months'
+      },
+      {
+        id: UnderservedPopulationEnum.ACCESSTOCARECANNOTOBTAINORPAYFOR,
+        label: 'Cannot easily obtain or pay for medical care'
+      }
+    ]
+  },
+  {
+    title: 'Education',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.EDUCATIONINCOMELESSTHANHIGHSCHOOLGRADUATE,
+        label: 'Less than a high school graduate'
+      },
+      {
+        id: UnderservedPopulationEnum.EDUCATIONINCOMELESSTHANTWENTYFIVETHOUSANDFORFOURPEOPLE,
+        label: 'Less than $25,000 for 4 people'
+      }
+    ]
+  }
+];
+
+export class WorkspaceUnderservedPopulation extends
+    React.Component<{value: Array<UnderservedPopulationEnum>, onChange: Function},
+    {show: boolean}> {
+  constructor(props: any) {
+    super(props);
+    this.state = {show: false};
+  }
+
+  flipShow() {
```

Comment:
Nit: This could probably just be inlined? I'm not sure it needs to be its own function.